src/ontology/edits/oeo-physical.omn

```diff
@@ -1787,6 +1792,7 @@ Class: OEO_00000072
 
     Annotations: 
         <http://purl.obolibrary.org/obo/IAO_0000115> "A biofuel is a combustion fuel that has a biogenic origin.",
+        <http://purl.obolibrary.org/obo/IAO_0000118> "biogener Brennstoff"@de,
         <http://purl.obolibrary.org/obo/IAO_0000118> "biogenic combustion fuel",
```

Comment:
```suggestion
        <http://purl.obolibrary.org/obo/IAO_0000118> "biogenic combustion fuel"@en,
```
At least for those classes where we now add German alternative terms, the English alternative terms should get the `@en` marker.